.github/workflows/deploy-oss-catalog.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy OSS Connector Catalog to GCS
+
+on:
+  push:
+    branches:
+      - master
+    paths: 
+      - airbyte-config/init/src/main/resources/seed
```

Comment:
nice - only runs when the right directory has been changed, only on the master branch.